### PR TITLE
Update finding

### PR DESCRIPTION
**Pull request by torgo**
This is a first pass PR at updating our finding on third party cookies to use stronger wording and remove text that and examples that have become outdated.

Adresses #21

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,7 +33,7 @@
 
 <section id="abstract">
 
-This finding discusses the deprecation of third-party (<abbr title="also known as">AKA</abbr> cross-site) cookies, the reasons for restricting them, and the challenges in removing them from the web platform. We highlight some use cases that depend on third-party cookies and offer some examples of designed-for-purpose technologies that can replace them. Specification authors are expected to ensure they do not undermine the benefits of removing third-party cookies when proposing new web platform technologies.
+This finding discusses why third-party (<abbr title="also known as">AKA</abbr> cross-site) cookies cookies are harmful to the web, why they must be deprecated, and the challenges in removing them from the web platform. We highlight some use cases that depend on third-party cookies and offer some examples of designed-for-purpose technologies that can replace them. Specification authors are expected to ensure they do not undermine the benefits of removing third-party cookies when proposing new web platform technologies.
```
Comment (hadleybeeman) [inline index.html]:
```suggestion
Third-party (<abbr title="also known as">AKA</abbr> cross-site) cookies are harmful to the web, and must be removed from the web platform. This finding explains why they must be deprecated, and examines the challenges in removing them. We highlight some use cases that depend on third-party cookies and offer some examples of designed-for-purpose technologies that can replace them. Specification authors are expected to ensure they do not undermine the benefits of removing third-party cookies when proposing new web platform technologies.
```

Comment (hadleybeeman) [reply index.html]: I'm trying to punch this up a bit, make the statement stronger and easier for others to quote

File: index.html [hunk 2]
```diff
@@ -42,7 +45,7 @@
...
 
-Many browsers are restricting or completely deprecating the use of of third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/), [Chrome](https://blog.google/products/chrome/updated-timeline-privacy-sandbox-milestones/)).  The TAG supports efforts to deprecate third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
+Many browsers are restricting or completely deprecating the use of of third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortuntetly, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG supports efforts to deprecate third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```
Comment (rhiaro) [inline index.html]:
```suggestion
Many browsers are restricting or completely deprecating the use of of third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG supports efforts to deprecate third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```

File: index.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>Improving the web without third-party cookies</title>
+<title>Third Party Cookies Must Be Deprecated</title>
```
Comment (rhiaro) [inline index.html]: :+1:

**Review by rhiaro — APPROVED** [review]
Looks good, with Hadley's suggested changes

File: index.html [hunk 4]
```diff
@@ -12,6 +12,9 @@
       url: "https://rhiaro.co.uk",
       company: "Digital Bazaar",
     },
+    {
+      name: "Daniel Appelquist"
```
Comment (torgo) [inline index.html]:
```suggestion
      name: "Daniel Appelquist",
      company: "W3C Invited Expert",
```

File: index.html [hunk 5]
```diff
@@ -12,6 +12,10 @@
       url: "https://rhiaro.co.uk",
       company: "Digital Bazaar",
     },
+    {
+      name: "Daniel Appelquist",
+      company: "W3C Invited Expert",
+    }
```
Comment (torgo) [inline index.html]:
```suggestion
    },
```

File: index.html [hunk 6]
```diff
@@ -62,15 +66,14 @@
...
+The TAG considers each new technology proposal *both* individually, *and* as they fit together with the web platform as a whole. The web must be cross-platform, so multi-implementer (multi-browser) support and developer support for privacy-related specifications is essential if they are going to achieve the goal of increasing privacy on the web. When we consider whether something makes the web platform better, we should be explicit about what the baseline for comparison is. Is a proposal better for privacy when compared to usage of third-party cookies? Or when compared with a web free from third-party cookies altogether? What about when some user agents restrict third-party cookies, but others do not?
```
Comment (martinthomson) [inline index.html:80]: Why this change?  It doesn't seem relevant to the main thrust of the change (and I think that it's a regression).

File: index.html (hunk 3 above)
Comment (martinthomson) [inline index.html]: The word "deprecated" is a poor choice here.

"Deprecated" is a state where something is not removed, but discouraged.  This is not what we should be saying.

```suggestion
<title>Third Party Cookies Are Antithetical To Privacy</title>
```

```suggestion
<title>Third Party Cookies Are Incompatible With Privacy</title>
```


```suggestion
<title>Third Party Cookies Are Bad</title>
```

Comment (torgo) [reply index.html]: How about:
```suggestion
<title>Third Party Cookies Considered Harmful</title>
```

File: index.html (hunk 6 above)
Comment (torgo) [reply index.html:80]: Clears up the language around implementer vs stakeholder. I've had feedback from many quarters that when we mean "multi implementer" we should say implementer rather than simply "stakeholder". So I was going to say "multi-implementer and multi-stakeholder" but I realised iit would be clear to say "multi-implementer and developer"...

File: index.html (hunk 3 above)
Comment (hadleybeeman) [inline index.html]: What about this? I'm going for clean and clear.
```suggestion
<title>Third party cookies must be removed from the web</title>
```

Comment (hober) [reply index.html]: I prefer any of these four options to "Must Be Deprecated." Some thoughts:

Martin's three suggestions do not suggest a solution, they "merely" point out a problem. "Must Be Deprecated" was at least a call to action, if a call to the wrong specific action.

Considered Harmful is cliché, and it's not even *our* cliché, it's the IETF's. But as a term of art or whatever it does hit pretty hard to my ears. I wonder if it hits as hard as I think it does to an audience not as steeped in standards stuff.

I think the option that would go hardest would be something like:

```suggestion
<title>Third Party Cookies Are Incompatible With Privacy and Must Be Removed</title>
```

File: index.html [hunk 7]
```diff
@@ -30,7 +38,7 @@
 
 <section id="abstract">
 
-This finding discusses the deprecation of third-party (<abbr title="also known as">AKA</abbr> cross-site) cookies, the reasons for restricting them, and the challenges in removing them from the web platform. We highlight some use cases that depend on third-party cookies and offer some examples of designed-for-purpose technologies that can replace them. Specification authors are expected to ensure they do not undermine the benefits of removing third-party cookies when proposing new web platform technologies.
+Third-party (<abbr title="also known as">AKA</abbr> cross-site) cookies are harmful to the web, and must be removed from the web platform. This finding explains why they must be deprecated, and examines the challenges in removing them. We highlight some use cases that depend on third-party cookies and offer some examples of designed-for-purpose technologies that can replace them. Specification authors are expected to ensure they do not undermine the benefits of removing third-party cookies when proposing new web platform technologies.
```
Comment (hober) [inline index.html]:
```suggestion
Third-party (<abbr title="also known as">AKA</abbr> cross-site) cookies are harmful to the web, and must be removed from the web platform. This finding explains why they must be removed, and examines the challenges in removing them. We highlight some use cases that depend on third-party cookies and offer some examples of designed-for-purpose technologies that can replace them. Specification authors are expected to ensure they do not undermine the benefits of removing third-party cookies when proposing new web platform technologies.
```

File: index.html [hunk 8]
```diff
@@ -42,17 +50,17 @@
...
 
-Many browsers are restricting or completely deprecating the use of of third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/), [Chrome](https://blog.google/products/chrome/updated-timeline-privacy-sandbox-milestones/)).  The TAG supports efforts to deprecate third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
+Many browsers are restricting or completely deprecating the use of of third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG supports efforts to deprecate third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```
Comment (hober) [inline index.html]:
```suggestion
Many browsers are restricting, deprecating, or completely removing support for third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG supports efforts to drop support for third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```

File: index.html [hunk 9]
```diff
@@ -42,17 +50,17 @@
...
 
 Removing third-party cookies from the web platform is not without complications. There are use cases for third-party cookies that need to be preserved, and pitfalls we need to be careful to avoid while doing so. This document sets out some things that specification editors and implementors should be aware of in order to make sure we ultimately [leave the web better than we found it](https://www.w3.org/TR/design-principles/#leave-the-web-better) after third-party cookies are deprecated.
 
-## Why restrict third party cookies?
+## Why deprecate third party cookies?
```
Comment (hober) [inline index.html]:
```suggestion
## Why remove third party cookies?
```

File: index.html [hunk 10]
```diff
@@ -42,17 +50,17 @@
...
+Many browsers are restricting or completely deprecating the use of of third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG supports efforts to deprecate third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
 
 Removing third-party cookies from the web platform is not without complications. There are use cases for third-party cookies that need to be preserved, and pitfalls we need to be careful to avoid while doing so. This document sets out some things that specification editors and implementors should be aware of in order to make sure we ultimately [leave the web better than we found it](https://www.w3.org/TR/design-principles/#leave-the-web-better) after third-party cookies are deprecated.
```
Comment (hober) [inline index.html]:
```suggestion
Removing third-party cookies from the web platform is not without complications. There are use cases for third-party cookies that need to be preserved, and pitfalls we need to be careful to avoid while doing so. This document sets out some things that specification editors and implementors should be aware of in order to make sure we ultimately [leave the web better than we found it](https://www.w3.org/TR/design-principles/#leave-the-web-better) after third-party cookies are removed.
```

File: index.html [hunk 11]
```diff
@@ -62,15 +70,14 @@
...
 * [CHIPS](https://github.com/w3ctag/design-reviews/issues/654). The goal of CHIPS is to allow state, without supporting correlation or tracking between web sites which don't know they are collaborating, for example, when embedding third-party services like customer service webchat.
-* [Fenced Frames](https://github.com/w3ctag/design-reviews/issues/735). This provides a new top-level browsing context which enables content from multiple parties to be embedded on a page, without the embedding site being able to communicate with the embedded site.
 
 In all cases, it is important to continually review how such proposals might interact with other emerging or proposed APIs. Be aware that a set of new technologies which carry minimal risk individually, could be used in combination for tracking or profiling of web users.
 
 ## Leaving the web better than we found it
 
 We support deprecating third-party cookies from the web platform, and we embrace the opportunity to improve the privacy features of the web. When we review new technologies to replace third-party cookies, we need to ensure that the replacements do not recreate the same pitfalls to privacy.
```
Comment (hober) [inline index.html]:
```suggestion
We support removing third-party cookies from the web platform, and we embrace the opportunity to improve the privacy features of the web. When we review new technologies to replace third-party cookies, we need to ensure that the replacements do not recreate the same pitfalls to privacy.
```

**Review by hober — CHANGES_REQUESTED** [review]
I really like the general direction of this update.

**Comment by hadleybeeman** [comment]
I'm fine with all of @hober's suggestions.

File: index.html (hunk 3 above)
Comment (martinthomson) [reply index.html]: I'm good with that.  I did not suggest "considered harmful" as I am of the opinion that that title should be considered harmful.

**Review by hadleybeeman — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (hadleybeeman) [reply index.html]: Sorry to be fussy, but "incompatible with privacy" wasn't in the title I was proposing (but it is in the version I see in the Files Changed tab). It may be true — and I won't die on this hill — but editorially, it's a lot clunkier.

File: index.html [hunk 12]
```diff
@@ -62,15 +70,14 @@
 
 * [FedCM](https://github.com/w3ctag/design-reviews/issues/718). Third-party cookies have been used to support single sign-on. FedCM is a set of technologies built to support identity federation, without replicating all functionality of third-party cookies.
 * [CHIPS](https://github.com/w3ctag/design-reviews/issues/654). The goal of CHIPS is to allow state, without supporting correlation or tracking between web sites which don't know they are collaborating, for example, when embedding third-party services like customer service webchat.
```
Comment (martinthomson) [inline index.html:72]: People keep saying that CHIPS is a replacement worthy of mention here.  It is not.

Yes, that leaves us with one example.  That's OK.  I suggest that we restructure around a single example instead.

File: index.html [hunk 13]
```diff
@@ -62,15 +70,14 @@
...
+The TAG considers each new technology proposal *both* individually, *and* as they fit together with the web platform as a whole. The web must be cross-platform, so multi-implementer (multi-browser) support and developer support for privacy-related specifications is essential if they are going to achieve the goal of increasing privacy on the web. When we consider whether something makes the web platform better, we should be explicit about what the baseline for comparison is. Is a proposal better for privacy when compared to usage of third-party cookies? Or when compared with a web free from third-party cookies altogether? What about when some user agents restrict third-party cookies, but others do not?
 
 Many varied proposals are being incubated in W3C Community Groups (eg. [PATCG](https://patcg.github.io/), [Privacy CG](https://github.com/privacycg/), [WICG](https://wicg.io/)) as well as outside (eg. [Privacy Sandbox](https://www.privacysandbox.com/)), and in these incubation stages multi-stakeholder support, consensus, and possible timelines for standardization are uncertain, and far from guaranteed.
```
Comment (martinthomson) [inline index.html:82]: Should this mention PATCG?

Comment (martinthomson) [inline index.html:82]: There are several other places that the word "deprecate", or variations thereof, appears.  Line 77 (left) or 84 (right) has another.

**Review by martinthomson — COMMENTED** [review]
I'm OK with the slightly clunky heading.  It's better than what @hadleybeeman suggests.

File: index.html (hunk 13 above)
Comment (hober) [reply index.html:82]: It does. Are you suggesting it shouldn't, Martin?

File: index.html (hunk 12 above)
Comment (hober) [reply index.html:72]: I'm fine with keeping or dropping the CHIPS example, but I'd love to hear more about why you want to drop it.

Comment (torgo) [reply index.html:72]: I'd really prefer to have at least 2 examples of what we're talking about. Do you strongly object to listing CHIPS? Would you be ok with listing CHIPS but hedging the language a bit?

File: index.html (hunk 13 above)
Comment (torgo) [reply index.html:82]: Maybe? Could you suggest some text?

Comment (hober) [reply index.html:82]: I thought I caught all of them. Did I miss some?

**Review by hober — APPROVED** [review]
Looks good to me.

File: index.html (hunk 13 above)
Comment (torgo) [reply index.html:82]: I don't think we need to wipe the word deprecate from the document.

File: index.html [hunk 14]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>Improving the web without third-party cookies</title>
+<title>Third Party Cookies Are Incompatible With Privacy and Must Be Removed</title>
```
Comment (jyasskin) [inline index.html]: +1 to @hadleybeeman (https://github.com/w3ctag/web-without-3p-cookies/pull/22#discussion_r1688647375) that

```suggestion
<title>Third Party Cookies Must Be Removed</title>
```

is punchier. (Or even "Third Party Cookies delenda est" ;) But no real objections to the current proposed text.

File: index.html [hunk 15]
```diff
@@ -42,17 +50,17 @@
...
 
-Many browsers are restricting or completely deprecating the use of of third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/), [Chrome](https://blog.google/products/chrome/updated-timeline-privacy-sandbox-milestones/)).  The TAG supports efforts to deprecate third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
+Many browsers are restricting, deprecating, or completely removing support for third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG supports efforts to drop support for third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```
Comment (jyasskin) [inline index.html]: The "or completely removing" is misleading: everyone has the Storage Access API, which preserves 3p cookies, and nobody has any plans to remove it. A possible fix, although I'm not tied to the exact wording:

```suggestion
Many browsers have restricted and/or deprecated third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG supports efforts to drop support for third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```

File: index.html (hunk 13 above)
Comment (jyasskin) [reply index.html:82]: We should ensure that every use of "deprecate" means just "discourage use of", and when we mean "remove", we should say "remove". I see 3 uses that should probably be replaced in the "Leaving the web better" section, including the one Martin mentioned.

**Review by jyasskin — COMMENTED** [review]
I won't be able to check in very often while I'm on vacation, so please don't block on me, but I wanted to leave a couple comments.

Overall, I like the change to set a firmer direction toward the eventual removal of 3p cookies

I think we're missing some text to explain why the Storage Access API was a good thing to add even though it preserves 3p cookies, and even though we don't know of a viable path to remove it again. Since this Finding is likely to be referred to in design reviews, folks should have some indication that interim APIs like that, and future ones that perhaps integrate Storage Access into other purpose-built APIs, can still get a positive reception. Or if folks think bridging APIs like that were a bad idea, this document should make that case. If there's no TAG consensus either way on that question, that should at least be explicit.

File: index.html (hunk 15 above)
Comment (plinss) [inline index.html]: Given the title change, maybe we should change "The TAG supports efforts to drop support for third-party cookies" to something more direct.

```suggestion
Many browsers are restricting, deprecating, or completely removing support for third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG insists that all browsers drop support for third-party cookies, as this improves the privacy preserving features of the web platform.
```

Comment (torgo) [reply index.html]:
```suggestion
Many browsers have restricted and/or deprecated third-party cookies (see: [Webkit](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), [Mozilla](https://blog.mozilla.org/en/mozilla/firefox-rolls-out-total-cookie-protection-by-default-to-all-users-worldwide/). Unfortunately, [not all browsers have followed suit](https://privacysandbox.com/intl/en_us/news/privacy-sandbox-update/). The TAG calls for all browsers to drop support for third-party cookies, as this provides an opportunity to further improve the privacy preserving features of the web platform.
```

**Review by rhiaro — APPROVED** [review]
